cmd/hatchet-engine/engine/run.go

```diff
@@ -39,6 +41,16 @@ func Run(ctx context.Context, cf *loader.ConfigLoader) error {
 
 	var teardown []Teardown
 
+	h := health.New()
+	healthProbes := sc.HasService("health")
```

Comment:
I figured I also create a service for this. Not sure if this is needed but I figured everything else is configured as a service. Also not sure if there is any other place where I need to put this.